stacks/udi/devfile.yaml

```diff
@@ -0,0 +1,29 @@
+schemaVersion: 2.2.0
+metadata:
+  name: udi
+  displayName: Universal Developer Image
+  description: Universal Developer Image provides various programming languages tools and runtimes for instant coding
+  icon: https://landscape.cncf.io/logos/devfile.svg
+  tags:
+    - Java
+    - Maven
+    - Scala
+    - PHP
+    - .NET
+    - Node.js
+    - Go
+    - Python
+    - Pip
+    - ubi8
+  projectType: default
+  language: Polyglot
+  version: 1.0.0
+components:
+  - name: tools
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-latest
```

Comment:
We usually recommend a specific version of the image instead of "latest" for stability reasons. Just in case the latest image got changed that affects the function of the devfile. Is there a specific version that we can use?

Reply:
@l0rd wdyt, as I recall we want to stick to latest

Reply:
This image has a lot of things in different versions and in some cases there can be even multiple versions of the same thing. 
A versioning image beyond UBI8 probably wouldn't make much sense. I think that it is actually good to have `latest` in this case. As I understand UDI users are expected to manage versions themself using things like SDKMan, nvm, virtuelenv, etc....

Reply:
ok, we can leave it as latest.